src/System.Private.CoreLib/shared/System/Resources/ResourceManager.cs

```diff
@@ -250,6 +233,8 @@ public ResourceManager(string baseName, Assembly assembly, Type usingResourceSet
                 throw new ArgumentException(SR.Arg_ResMgrNotResSet, nameof(usingResourceSet));
             _userResourceSet = usingResourceSet;
 
+            SetAppXConfiguration();
+
```

Comment:
This SetAppXConfiguration call was missing in CoreCLR, but present in CoreRT. It may have been intentional, but in that case it should still be here with `if (_userResourceSet == null)`. Certainly the calls to SetAppXConfiguration are inconsistent.